phpstan-baseline.neon

```diff
@@ -1,5 +1,8 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Cannot call method (log|debug|info|notice|warning|error|critical|alert|emergency)\\(\\) on Psr\\\\Log\\\\LoggerInterface\\|null\\.$#"
+
```

Comment:
Move this section to `phpstan.neon`. It does not belong to `phpstan-baseline.neon`, because this file is generated automatically and this will be overwritten.